docs/advanced-entity-analytics/entity-store.asciidoc

```diff
@@ -0,0 +1,52 @@
+[[entity-store]]
+= Entity store
+
+preview::[]
+
+.Requirements
+[sidebar]
+--
+To use the entity store, you must have the appropriate privileges. For more information, refer to <<ers-requirements, Entity risk scoring requirements>>.
+--
+
+The entity store allows you to query, reconcile, maintain, and persist entity metadata from various sources such as:
+
+* Ingested log sources
+* Integrated identity providers (such as Active Directory, EntraID, and Okta)
+* Internal and external alerts
+* External asset repositories
+* Asset criticality submissions
+
+The entity store can store any entity type observed by {elastic-sec}. This allows you to store, view, and query select entities directly from an index without needing to perform real-time searches of observable data. The entity store extracts entities from all indices in the {elastic-sec} <<default-data-view-security, default data view>>.
+
+When the entity store is enabled, the following resources are generated:
```

Comment:
```suggestion
When the entity store is enabled, the following resources are generated for each stored entity:
```
Not sure if this is accurate but might be nice to specify what the generated resources are for

Reply:
I like the general suggestion, instead of "for each stored entity" we can instead say something like "for hosts and users", as the resources are created for each entity _type_, not each individual entity